Leanplum-SDK/Classes/LPRequestStorage.m

```diff
@@ -98,8 +98,8 @@ - (void)migrateRequests
 - (void)saveRequests:(NSMutableArray *)requests
 {
     if (requests.count > MAX_STORED_API_CALLS) {
-        NSRange range = NSMakeRange(requests.count - MAX_STORED_API_CALLS, MAX_STORED_API_CALLS);
-        requests = [[requests subarrayWithRange:range] mutableCopy];
+        NSRange range = NSMakeRange(0, requests.count - MAX_STORED_API_CALLS);
```

Comment:
shouldn't this be sth like:

NSRange range = NSMakeRange(requests.count - MAX_STORED_API_CALLS, MAX_STORED_API_CALLS);